CompanyManager.DAL/Repository/PersonService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompanyManager.DAL.Repository
+{
+    class PersonService
```

Comment:
Klasę bym zrobił public, bo będzie używana w innym projekcie, a tak to domyślnie ma zakres internal, czyli pewnie będzie dostępna dla tylko tego projektu.